install/lng/german.lng.php

```diff
@@ -63,6 +63,7 @@
 $lng['install']['admin_pass2'] = 'Administrator-Passwort (Bestätigung)';
 $lng['install']['activate_newsfeed'] = 'Aktiviere das offizielle Newsfeed<br><small>(https://inside.froxlor.org/news/)</small>';
 $lng['install']['serversettings'] = 'Servereinstellungen';
+$lng['install']['distribution'] = 'Verteilung';
```

Comment:
'Verteilung' ist the direct translation of distribution, you can safely use "Distribution" is german too

Reply:
ah ok.